features-support/step_definitions/features.js

```diff
@@ -24,11 +24,27 @@ function getFeaturesFromUrl(callback) {
     });
 }
 
+var staticTestDataExists = false;
+
 module.exports = function () {
+
+  // Create static test data, only do this once.
   this.Given(/^a set of specifications containing at least one feature file\.?$/, function (callback) {
     var world = this;
-    world.createSpecsForTesting()
+
+    // Only generate the static test data once.
+    if (staticTestDataExists) {
```

Comment:
`@cleanSlate` ensures that test data is removed before that scenario runs. This step only deletes and creates test data the first time it is run (state is tracked via closure outside of the step) so if five scenarios use it we still only write the test data once.